railgun/config/samples/configuration.konghq.com_v1beta1_tcpingress.yaml

```
@@ -0,0 +1,7 @@
+apiVersion: configuration.konghq.com.konghq.com/v1beta1
+kind: TCPIngress
+metadata:
+  name: tcpingress-sample
+spec:
+  # Add fields here
+  foo: bar
```

This looks very much a stub that doesn't actually contain a TCPIngress example. Maybe not much of an issue since I don't know that we use these examples anywhere yet, but given that UDPIngress has a realistic example already, this should also. Copied from the existing [docs site example](https://docs.konghq.com/kubernetes-ingress-controller/1.2.x/guides/using-tcpingress/#tcp-port-based-routing)
```suggestion
spec:
  rules:
  - port: 9000
    backend:
      serviceName: tcp-echo
      servicePort: 2701```